ios/NewExpensify/OriginImageRequestHandler.h

```diff
@@ -0,0 +1,12 @@
+//
+//  OriginImageRequestHandler.h
+//  NewExpensify
+//
+//  Created by ntdiary on 2022/8/8.
+//
+
+#import <React/RCTURLRequestHandler.h>
+
+@interface OriginImageRequestHandler : NSObject <RCTURLRequestHandler>
+
+@end
```

Comment:
Why this file?

Reply:
This is just a header file created by convention.